core/model/modx/processors/resource/update.class.php

```diff
@@ -452,6 +452,18 @@ public function checkForUnPublishOnSiteStart() {
      * @return boolean
      */
     public function checkDeletedStatus() {
+	    if (1 == $this->getProperty('deleted', null)) {
+	    	if ($this->modx->getOption('site_start') == $this->getProperty('id')) {
```

Comment:
That's mostly correct @Jako - `$modx->getOption()` will indeed get a context setting, but in the manager (and by extension in most processors), the context is `mgr`, not the working context derived from the resource which is what's needed.